src/http-client.ts

```diff
@@ -41,7 +41,7 @@ export const fetchUnfucked = async (
   }
   const response = await fetch(uri, options);
 
-  let contentType = CONTENT_TYPES.JSON;
+  let contentType: string | undefined = CONTENT_TYPES.JSON;
```

Comment:
No `undefined`. It's always a string and it's default type is the content type for JSON.

Reply:
it gets overwriten in line 50 and the .at(0) can return undefined

https://github.com/onsol-labs/solana-wallet-names/blob/7996db0a0b0e7b346a4e725402e926ac75c285a8/src/http-client.ts#L50

Reply:
I added it so TS wouldn't not complain it being an error